### PR TITLE
hw01 _ Shuo

### PR DESCRIPTION
**Pull request by shuoq25**
Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

### CODE REVIEW

**Comment by shuoq25** [comment]
It is not so challenging in forking the repo, downloading the clone, and editing the readme.md.  
At first, I am not very used to writing in Markdown format, even with the help of the cheatsheet. I still don't know how to resize the photo, adjust the position of texts, or adjust the font; these features  are easier to use in GUI workflow. Nevertheless, I start to realize the benefit of using RMarkdown -- it is simple to reproduce. I would love to learn more about R Markdown technique.

It is also confusing where I should submit this comment on hw01. Should I include it in the readme.md, the commit comment, or this comment?

**Comment by bensoltoff** [comment]
@shuoq25 submit your self-reflection in the comment field when you submit the pull request

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Self-Introduction
+### Basic Information
+Hi, my name is Shuo Qiu, a third-year college student majoring in Sociology. Sociology is interesting to me because it can explain collective phenomenon that a study of only individuals cannot.
```
Comment (bensoltoff) [inline README.md:5]: I recommend leaving a blank line between different Markdown elements, such as between headers and paragraph text. It avoids rendering issues in more complex documents

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+Hi, my name is Shuo Qiu, a third-year college student majoring in Sociology. Sociology is interesting to me because it can explain collective phenomenon that a study of only individuals cannot.  
+
+### Hobbies
+Besides the academic intersts, I have several hobbies:
+
+* Play piano
+  + my favorite song is *The Silver Clouds Chasing the Moon*
+* Play basketball
+  + I built a basketball team in my second year and played in the intramurals. Here is a photo I took with my teammates in the Crown Gym before a game:
+![Me and my Teammates](WechatIMG15035.jpeg)
```
Comment (bensoltoff) [inline README.md:14]: Nice addition of an image you uploaded in the repo

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        x   |              |            |
| **Coding strategy**         |        x   |              |            |
| **Presentation**            |         x  |              |            |
| **Achievement, creativity** |           |   x           |            |
| **Reproducibility**         |       x    |              |            |

**Comment by sabreena-croteau** [comment]
I think that you did a great job on the headers and the list. It made it very clear. I'm not sure if it was just the way I viewed it on git but the image didn't come through.

They put friends back up?!

**Pull request closed without merging**